Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/DriveBase.java

```diff
@@ -70,36 +78,46 @@ public DriveBase() {
 
 		leftFrontMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
 		rightFrontMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
-		// leftBackMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
-		// rightBackMotor.configOpenloopRamp(secondsFromNeutral, timeoutMs);
+
+		// TODO: Look at the manual for "Follower" to see if we need to set the open
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)